### PR TITLE
AzureStack Commerce Admin SDK Regeneration

### PR DESCRIPTION
**Pull request by deathly809**
## Description

An internal review was performed since the last commit and the old spec was thrown out and a new one was generated.

Accepted spec PR
https://github.com/Azure/azure-rest-api-specs/pull/2777

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/AzureStack/CommerceAdmin/Commerce.Admin/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -24,5 +24,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.1.1.0")]
```
Comment (dsgouda) [inline src/AzureStack/CommerceAdmin/Commerce.Admin/Properties/AssemblyInfo.cs]: Please don't update the `AssemblyVersion`

File: src/AzureStack/CommerceAdmin/Commerce.Admin/generate.cmd [hunk 2]
```diff
@@ -4,6 +4,7 @@
 ::
 
 @echo off
-call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/commerce latest Azure current azure-rest-api-specs %CD%
+call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/commerce latest Azure bcb638a668dbb79eaf3190324ca6d690be84b9f6 azure-rest-api-specs %CD%
```
Comment (dsgouda) [inline src/AzureStack/CommerceAdmin/Commerce.Admin/generate.cmd]: Please create a `generate.ps1` file like [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.
This should also generate/modify a .txt file, please commit those changes

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/AzureStack/CommerceAdmin/Commerce.Admin/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (deathly809) [reply src/AzureStack/CommerceAdmin/Commerce.Admin/Properties/AssemblyInfo.cs]: Reverted.

File: src/AzureStack/CommerceAdmin/Commerce.Admin/generate.cmd (hunk 2 above)
Comment (deathly809) [reply src/AzureStack/CommerceAdmin/Commerce.Admin/generate.cmd]: Done.

**Comment by deathly809** [comment]
@dsgouda  Changes made.

File: src/AzureStack/CommerceAdmin/Commerce.Admin/generate.ps1 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Remove old
+Remove-Item -Path "$PSScriptRoot\Generated" -Force -Recurse
```
Comment (dsgouda) [inline src/AzureStack/CommerceAdmin/Commerce.Admin/generate.ps1]: Why did you have to do this?

Comment (dsgouda) [reply src/AzureStack/CommerceAdmin/Commerce.Admin/generate.ps1]: I think I know why, like most other RPs/Language sdks you should really be setting the output-folder and other csharp settings in the configuration file [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/commerce/resource-manager/readme.md)
This is not a blocking requirement so please remove this file and regenerate using the .cmd for now.
We highly recommend updating the configuration file though.

Comment (deathly809) [reply src/AzureStack/CommerceAdmin/Commerce.Admin/generate.ps1]: Can I just leave it as it is for now until I update the spec to change the output folder.  Once I do that I can do another PR to update the generate.ps1.

Comment (dsgouda) [reply src/AzureStack/CommerceAdmin/Commerce.Admin/generate.ps1]: That works, please create an action item at your end to this effect.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**